src/pages/solutions/for-students.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: We help students 
+quickLinks:
+  - name: Data
+    anchor: data
+  - name: Basemaps and Imagery
```

Comment:
there's a discussion in the chat to continue using two words for base maps

Reply:
old habits die hard.